test/byContract/IexecPoco/06_claim.test.ts

```diff
@@ -172,11 +173,9 @@ describe('Poco', async () => {
         expect((await iexecPoco.viewTask(taskId)).status).to.equal(TaskStatusEnum.REVEALING);
         await time.setNextBlockTimestamp(startTime + maxDealDuration);
 
-        const claimReceipt = await iexecPoco
-            .connect(anyone)
-            .claim(taskId)
-            .then((tx) => tx.wait());
-        expect(claimReceipt)
+        const claimTx = await iexecPoco.connect(anyone).claim(taskId);
```

Comment:
```suggestion
        const claimTx = await iexecPocoAsAnyone.claim(taskId);
```
in common for all calls.